app/application.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var root = 'http://localhost:4000/api';
+var origin = window.location.origin;
+
+var App = {
+  init: function init() {
+    $("body").prepend(
+        `<a href = ${origin}/new.html> + </a>`)
+  },
+  constableApi: class ConstableApi {
+
+    getAnnouncements(){
+      $.ajax({
+        url: root + '/announcements',
```

Comment:
What do you think about extracting an object that manages your paths? `path.announcments`